content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,81 +118,138 @@ On your validator:
+**Cons:**
 
-    **Warning:** Be sure that you don't publish your validator's IP address in other ways.
+  - Doesn't allow you to select your validator's peers, which means that you have no idea whether your peers may decide to act maliciously. Your validator is designed to protect itself against malicious peers, but avoiding direct contact with unknown peers is the safest configuration possible.
 
-        [peer_private]
-        1
+  - May connect you to peers that disconnect and change frequently, depending on whether you manage to connect to any reliable ones.
 
-    With `[peer_private]` enabled, `rippled` ignores any connections suggested by the `[ips]` stanza. If you need to connect to an IP currently in your `[ips]` stanza, put it in the `[ips_fixed]` stanza instead, but _**only**_ if you can expect them to behave as described in step 1.
+_**To connect your validator to the XRP Ledger network using discovered peers,**_ omit the `[peer_private]` stanza or set it to `0` in your validator's `rippled.cfg` file. The [example rippled.cfg file](https://github.com/ripple/rippled/blob/develop/cfg/rippled-example.cfg) is delivered with this configuration.
 
 
 ### Connect using proxies
 
-This configuration involves running stock `rippled` servers that you use as proxies between your validator and inbound and outbound network traffic.
+This configuration connects your validator to the network through stock `rippled` servers that you run yourself. These proxy servers sit between your validator and inbound and outbound network traffic.
 
 **Note:** While these servers are acting as proxies, they are not web proxies for HTTP(S) traffic.
 
-The benefit of this configuration is more redundancy and access to a lot of safe and reliable connections to the network through proxy servers that you run yourself. These connections help keep your validator healthy.
+**Pros:**
 
-<!-- { TODO: Future: add a recommended network architecture diagram to represent the proxy, clustering, and firewall setup: https://ripplelabs.atlassian.net/browse/DOC-2046 }-->
+  - Guarantees that these servers will try to maintain a connection with your validator.
+
+  - Provides your validator with connections to the network that are as redundant and reliable as you make them.
 
-1. [Enable validation](#3-enable-validation-on-your-rippled-server) on your `rippled` server.
+  - Enables you to trust your validator's peers.
 
-2. Set up stock `rippled` servers. For more information, see [Install rippled](install-rippled.html).
+  - Enables you to create as many direct peer connections as you want. Being able to request content from more peers enables your validator to parallelize the process of downloading the current ledger (when syncing) or backfilling history and gives it direct access to a wider selection of history.
```

Comment:
A validator should not store or serve history.

The current ledger is very likely served from cache at all your peers, so having a lot of peer connections is not going to help a lot there either - they all will serve that data as fast as they can and as fast as you request it already.